src/app/pages/merge-page/merge-page.component.html

```diff
@@ -0,0 +1,52 @@
+<app-actions-bar-merge></app-actions-bar-merge>
+<mat-progress-bar
+  mode="indeterminate"
+  color="primary"
+  *ngIf="pageState.loading"
+></mat-progress-bar>
+<ng-container *ngIf="!pageState.loading">
+  <div *ngIf="candidatesMatrixNotEmpty; else noCandidatesId" class="candidate-merge-container">
+    <div class="desc-titles-container">
+      <div class="desc-title"></div>
```

Comment:
why we need this div element?

Reply:
Empty element at the beginning of the table